Apps/WebClient/src/appsettings.Development.json

```diff
@@ -49,7 +49,7 @@
         "Laboratory": "http://localhost:3004/",
         "Medication": "http://localhost:3003/",
         "Patient": "http://localhost:3002/",
-        "Ticket": "http://localhost:8000/"
+        "Ticket": "https://tickets.healthgateway.gov.bc.ca/"
```

Comment:
Is this change intentional or should be reverted?

Reply:
I spoke with @sslaws.  This was intentional.  WaitingQueueWeb is a project outside of HealthGateway. WaitingQueueWeb will need to be brought in locally in order for local dev to work. This will be sorted out later.